framework_tvb/tvb/adapters/visualizers/surface_view.py

```diff
@@ -386,17 +387,17 @@ def _compute_measure_param(connectivity_measure, measure_points_no):
             max_measure = measure_points_no
             client_measure_url = ''
         else:
-            connectivity_measure_shape = connectivity_measure.array_data.shape
```

Comment:
On such methods, we could add `type` and specify that `connectivity_measure` param is of type `ConnectivitymeasureIndex`